policy/README.md

```diff
@@ -0,0 +1,12 @@
+Policies allow tenants to perform additional evaluation of attestation
+evidence that is not covered by a particular attestation scheme. The policy can
+be used to override the overall attestation status and/or the trust vector
+values in the result (e.g. rejecting a token considered valid by the scheme if
+the more stringent constraints described in the policy are not met).
+
```

Comment:
It may be worth adding a couple of lines to point out that handling of policy inputs (files?) will likely be subject to future administrative framework. The current system makes it part of the deployment development but interaction with 'policy admin' actors may follow.